check-licenses/action.yml

```diff
@@ -0,0 +1,68 @@
+name: "Check licenses"
+description: "Check if package licenses in the current environment are among the accepted ones"
+
+inputs:
+  python-version:
+    description: 'Desired Python version.'
+    default: '3.10'
+    required: false
+    type: string
+  use-python-cache:
+    description: 'Whether to use Python cache or not. Requires a pyproject.toml file.'
+    required: false
+    default: true
+    type: boolean
+  skip-install:
+    description: 'Skip project installation.'
+    default: false
+    required: false
+    type: boolean
+
```

Comment:
I left you a comment on the ``pyansys/pyansys`` PR. Please have a look at it: https://github.com/pyansys/pyansys/pull/97#issuecomment-1328592298

We might need to provide an additional optional argument called ``target``. Plan on implementing it.

Reply:
Done.